release_notes.md

```diff
@@ -5,3 +5,38 @@
 The file describes all the changes, that are affected the usage of the application.
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
+
+## [0.1.0] - [29.05.2023]
+
+### New features
+
+`Quicklook`: Users can check a list of pet sitters available for pet owners. Each list row includes an avatar, full name, price, short bio, and pet sitter rating.
+
+</details>
+
+<details>
+<summary> CLICK HERE</summary>
+
+<p>
+
+![list of pet sitters](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae)
```

Comment:
nit:
it is better to scale the screenshots so that they do not occupy the entire screen space in height
```suggestion
 <img src="https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae" width=50% height=50%>
```

Reply:
Thanks for useful comment!
I've deleted this screenshot, according to suggestion from Dersim.